frontend/packages/_plugin-style.scss

```diff
@@ -0,0 +1,4 @@
+/* Styles of all active plugins */
```

Comment:
Simple, but working.
See additional comments for more details.

Reply:
Can I ask for comments here, please? @spadgett , @vojtechszocs

Reply:
@mareklibra Sorry for responding late.

Plugin-specific styles can be loaded as a side effect of loading the plugin's entry module, e.g. in `kubevirt-plugin/src/plugin.ts`:

```ts
import './style.scss';
```

I see you've already updated the PR to do that, so :+1: on that.

Reply:
@mareklibra Please note, once #1708 is merged and you enable the `kubevirt-plugin` & run tests (`yarn test`), we'll need to ensure Jest properly mocks `scss` files. Right now, the relevant module mapper config is:

```
"\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
```

So we'll have to add `scss` there ^^.

Reply:
This is addressed by #1721 ([commit](https://github.com/openshift/console/pull/1721/commits/6db90c663cbdad13fb0e86b6d480c1829aab3ec5)).